plugins/org.python.pydev/tests_analysis/com/python/pydev/analysis/OccurrencesAnalyzerPy310Test.java

```diff
@@ -81,4 +81,310 @@ public void testMatchAndCaseAsSoftKeyword() {
                 + "print(result)");
         checkNoError();
     }
+
+    public void testMatchingMultiplePatterns() {
```

Comment:
It'd be nice to have some cases where the error is reported (in particular, in the `_` case, a test should be available to note that `_` is not defined).

Reply:
-- and if you think of other cases, those are welcome too ;)